.github/CODEOWNERS

```diff
@@ -290,6 +290,10 @@ sdk/ai/ai-inference-rest @glharper @dargilco @jhakulin
 # ServiceLabel: %Cognitive - Content Safety
 # AzureSdkOwners: @xirzec
 
+# PRLabel: %Operator Nexus - Network Cloud
```

Comment:
`Operator Nexus - Network Cloud` doesn't look like an existing label.  @ronniegeraghty

Reply:
@jeremymeng, the label was added to the repo last week. Rerunning the codeowners-linter job should pass now.